api/models/productDao.js

```diff
@@ -34,9 +47,9 @@ const getProductByCategoryId = async (categoryId) => {
                 p.discounted_price AS discountedPrice 
             FROM products AS p
             INNER JOIN sub_categories AS s
-            WHERE p.sub_category_id = s.id AND s.category_id = ?
-            `, [categoryId]
-        )
+            WHERE p.sub_category_id = s.id AND s.category_id = ${categoryId}
+            ${sortSet[price]}
+            `)
```

Comment:
조건문을 INNER JOIN 결합문으로 넣어주면 좋을 것 같아요~!

```suggestion
            INNER JOIN sub_categories AS s ON s.id = p.sub_category_id 
            WHERE s.category_id = ?
            ${sortSet[price]}
            `, [ categoryId ])
```